generators/app/templates/src/main/java/package/service/kafka/producer/EntityProducer.java.ejs

```diff
@@ -51,7 +57,11 @@ public class <%= entityClass %>Producer {
     }
 
     public void send(final <%= type %> message) {
+<%_ if (entityOrdered) { _%>
+        final ProducerRecord<String, <%= type %>> record = new ProducerRecord<>(topicName, <%= entityClass %>ProducerKey.<%= _.toUpper(entityClass) %>_READ.toString(), message);
```

Comment:
Why the enum "$entity_READ" is use by default, in the producer? 
it should be interesting to let the user define the key, because forcing the key value to an enum, can force producer to send all messages on an unique partition even if a topic is designed with multiple partitions.

Reply:
Because for now we are just reading on entities. We could maybe use the others when we'll link the whole entity CRUD to produce and consume events on these operations.

Reply:
I disagree with letting the user choose the key because it would end on too many prompt options and I would violate [policy 2 from JHipster](https://www.jhipster.tech/policies/).

Reply:
Ok i understand